package.json

```diff
@@ -39,14 +40,14 @@
   },
   "devDependencies": {
     "@uphold/github-changelog-generator": "^1.0.1",
-    "eslint": "^7.18.0",
-    "mocha": "^7.1.1",
+    "eslint": "^8.7.0",
+    "mocha": "^9.2.0",
     "pre-commit": "^1.2.2",
     "prettier": "^2.2.1",
     "should": "^9.0.2"
   },
   "peerDependencies": {
-    "eslint": "^7.18.0",
+    "eslint": "^8.7.0",
```

Comment:
```suggestion
    "eslint": ">=8",
```

Reply:
maybe we should do something similar to prettier to avoid warnings

Reply:
@Americas wdyt  ?

Reply:
I'm not clear what is actually using this TBH.

Reply:
Not sure what you mean. If you mean broadening the allowed versions, then yes, agree. If not, please explain.

Reply:
yes, maybe `>= 8 < 9`

Reply:
> `>= 8 < 9`

Isn't that equivalent to `^8`? 🤔

Reply:
I was going to say that 😂

Reply:
@waldyrious yes - you're right.

Reply:
```suggestion
    "eslint": "^8",
```